File: points_of_interest/v1/by_id/Python SDK/points_of_interest.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Give me information about a place based on it's ID
    '''
    response = amadeus.reference_data.locations.point_of_interest('9CB40CB5D0').get()
    print(response.data)
except ResponseError as error:
    raise error

File: points_of_interest/v1/get/pois_poisId/iOS SDK/PointsOfInterest.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.referenceData.locations.pointOfInterest(poiId: "9CB40CB5D0").get(
    onCompletion: {
        response, error in
        if error == nil {
            print(response!.data)
        }
    })
